src/main/java/ui/components/NavigableListView.java

```diff
@@ -111,25 +114,41 @@ public void restoreSelection() {
                 selectedIndex = Optional.of(nextIndex);
 
                 // The next item will be considered selected
-                onItemSelected.accept(nextIndex);
+                triggerItemSelected(nextIndex);
             }
         }
     }
 
+    /**
+     * Triggers the selected item on the card at the specified index for actions that are not right clicks
+     */
+    private void triggerItemSelected(int index){
+        onItemSelected.accept(index, !IS_RIGHT_CLICK);
+    }
+
+    /**
+     * Triggers the selected item on the card at the specified {index} for actions that are right clicks
```

Comment:
@codemanmeet I think the syntax you are looking for is `{@code index}` or something like that (can't remember the exact syntax, but should be easy to find if you look up javadoc documentation).